include/dt-bindings/pinctrl/esp32-pinctrl.h

```diff
@@ -9209,194 +9217,6 @@
 #define TWAI_TX_GPIO33 \
 	ESP32_PINMUX(33, ESP_NOSIG, ESP_TWAI_TX)
 
-/* UART0_CTS */
-#define UART0_CTS_GPIO0 \
```

Comment:
What is the reason for all RTS and CTS UART pins to be removed?

Reply:
@marekmatej @glaubermaroto Was this fixed? It seems to be remove from origina file.